src/insightsAPIBeaconClient.ts

```diff
@@ -13,6 +13,9 @@ export type InsightsApiEvent = {
   objectIDs?: string[];
   positions?: number[];
   filters?: string[];
+
+  appId?: string;
```

Comment:
same here right?

Reply:
I switched to providing additional headers directly, following Ben's comment.